development/docker-compose/docker-compose.yaml

```diff
@@ -19,7 +19,14 @@ x-common-variables: &common-variables
   management.security.roles: ANONYMOUS,ACTUATOR,USER
   management.endpoint.env.show-values: ALWAYS
   management.endpoint.configprops.show-values: ALWAYS
-  management.tracing.enabled: false
+  management.tracing.enabled: true
+  management.tracing.propagation.type: b3
```

Comment:
shouldn't we use w3c since it's default for spring boot 3 and clients normally use this nowadays